macros/multiple_databases/charindex.sql

```diff
@@ -0,0 +1,15 @@
+{%- macro charindex(expression_to_find, expression_to_search, start_location=None) -%}
+    {%- if start_location is none -%}
+        {%- if target.type == 'databricks' -%}
+            position('{{ expression_to_find }}', '{{ expression_to_search }}')
```

Comment:
No, on the contrary, it saves us from using extra escaping in the models. I.e., we can use
`{{ pm_utils.charindex('Approve purchase order', 'Purchase_order_item_event_log_with_internal_id."Activity"') }} > 0`
instead of
`{{ pm_utils.charindex("'Approve purchase order'", 'Purchase_order_item_event_log_with_internal_id."Activity"') }} > 0`
You could say this is redundant in the `expression_to_search` if we suppose that that is always a field in a table and not a hardcoded string, but we cover more use cases this way.

Reply:
If you compile now the code we are getting position('-', 'EKPO_input.`NETWR_input`')
while I believe we want position('-', EKPO_input.`NETWR_input`)
which you get if the macro implementation is position('{{ expression_to_find }}', {{ expression_to_search }})